### PR TITLE
Add homepage in doc site

### PR DESCRIPTION
**Pull request by abhisheksahani-nova**

### CODE REVIEW

File: Components/cards/cards.css [hunk 1]
```diff
@@ -31,7 +31,6 @@
   margin-top: 1rem;
   letter-spacing: 0.025rem;
   color: #838686;
```
Comment (Maianki) [inline Components/cards/cards.css]: use CSS variables for color.

File: Components/cosmicUI.css [hunk 2]
```diff
@@ -15,7 +17,8 @@ html {
 }
 
 body {
-  color: #555555;
+  color: #3E4152;
```
Comment (Maianki) [inline Components/cosmicUI.css]: Use CSS variables.

File: Doc.html/input.html [hunk 3]
```diff
@@ -8,38 +8,38 @@
     <link rel="stylesheet" href="../prism.css" />
     <link rel="stylesheet" href="../Components/cosmicUI.css" />
     <link rel="stylesheet" href="../Docs-css/Docs-main.css" />
+    <link rel="stylesheet" href="../style.css">
     <title>Cosmic UI | Input</title>
+    <script src="https://kit.fontawesome.com/cbde1196cc.js" crossorigin="anonymous"></script>
```
Comment (Maianki) [inline Doc.html/input.html]: use defer or include script at the end of the body.

File: Docs-css/Docs-navbar.css [hunk 4]
```diff
@@ -25,5 +34,9 @@
 }
 
 .nav-icon {
-  font-size: 1.5rem;
+  font-size: 1.4rem;
+}
+
+.docSite-nav-clr{
+  color: #3E4152;
```
Comment (Maianki) [inline Docs-css/Docs-navbar.css]: use CSS variable.

File: style.css [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+
+.homepage-content-container {
+  margin: 3rem;
+  padding: 2.5rem 4rem 4rem 9rem;
+}
+
+.homepage-title {
+  font-family: "Playfair Display", serif;
+  font-size: 6rem;
+  font-weight: 800;
```
Comment (Maianki) [inline style.css:13]: You are not importing a font with a weight of 800. So, I think it will not apply.

**Review by Maianki — CHANGES_REQUESTED** [review]
Suggested a few changes. Everything else looks good.

File: Components/cards/cards.css (hunk 1 above)
Comment (abhisheksahani-nova) [reply Components/cards/cards.css]: done

File: Components/cosmicUI.css (hunk 2 above)
Comment (abhisheksahani-nova) [reply Components/cosmicUI.css]: done

File: Doc.html/input.html (hunk 3 above)
Comment (abhisheksahani-nova) [reply Doc.html/input.html]: done

File: Docs-css/Docs-navbar.css (hunk 4 above)
Comment (abhisheksahani-nova) [reply Docs-css/Docs-navbar.css]: done

File: style.css (hunk 5 above)
Comment (abhisheksahani-nova) [reply style.css:13]: done

**Review by harshparab — APPROVED** [review]
Looks good to merge!

**Pull request merged**